app/app.go

```diff
@@ -297,6 +310,15 @@ func New(
 		appCodec, keys[ibchost.StoreKey], app.GetSubspace(ibchost.ModuleName), app.StakingKeeper, app.UpgradeKeeper, scopedIBCKeeper,
 	)
 
+	// register the proposal types
+	govRouter := govtypes.NewRouter()
+	govRouter.AddRoute(govtypes.RouterKey, govtypes.ProposalHandler).
+		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
+		// TODO: do we want a community pool at all? If not, does it even make sense to include that NewCommunityPoolSpendProposalHandler?
```

Comment:
Let's discuss the community pool in a separate issue/github dicsussion thread. I'll remove the todo after we have that issue/discusssion.